### ISSUE TITLE
spatialhashmap.go assignment mismatch

### ISSUE THREAD

**Issue by rpomykala** [psykhi/wordclouds#6]
Reproduction Steps:

```
$ go get -u github.com/psykhi/wordclouds
# github.com/psykhi/wordclouds
..\..\go\src\github.com\psykhi\wordclouds\spatialhashmap.go:36:5: assignment mismatch: 1 variable but uuid.NewV4 returns 2 values
```

Golang Version:

```
$ go version
go version go1.13.1 windows/amd64
```

**Comment by rpomykala** [comment]
https://github.com/psykhi/wordclouds/pull/7

**Comment by psykhi** [comment]
I think there's some conflicts between the versions of the UUID lib you have locally (probably from the master branch) and the one included in that repo (from the 1.2.0 tag https://github.com/satori/go.uuid/blob/f58768cc1a7a7e77a3bd49e98cdd21419399b6a3/generator.go). This repo uses go modules so I'd suggest not having it in your GOPATH.

**Comment by rpomykala** [comment]
Understandable, thanks!
